.github/workflows/CI.yml

```
@@ -58,9 +62,20 @@ jobs:
     with:
       environment: test
 
+  deploy_telegram_mini_app:
+    name: Deploy Telegram Mini App
+    if: ${{ github.event_name == 'push' }}
+    needs: [ build_push_backend, build_telegram_mini_app ]
+    permissions:
+      id-token: write # needed for aws credentials
+      contents: read
+    uses: ./.github/workflows/deploy-web-ui.yml
```

It looks like should be `deploy-telegram-mini-app`

```suggestion
    uses: ./.github/workflows/deploy-telegram-mini-app.yml
```

---

Yup, thanks